### PR TITLE
Implement Structure for BuildSnapshotContainer

### PR DESCRIPTION
**Pull request by samuelsogbesan**
This PR integrates the **basic structure for the Wrapper that will contain the renders of the BuildSnapshot Components**. It's illustrated visually in the image below. **This Component is known as "BuildSnapshotContainer"**.

The BuildSnapshotComponent **expects the buildbot data as an array in props**, and **maps each datapoint to an instance of the BuildSnapshot Component**. The Component wraps these BuildSnapshots in a **< div >**, represented by the < Wrapper > React Component / Abstraction.

_The red outline illustrates the dimensions of the component_. _Note the red line is **not** rendered on screen, and is just an illustration of intent._

image

### CODE REVIEW

File: src/main/webapp/components/BuildSnapshotContainer.js [hunk 1]
```diff
@@ -7,9 +9,10 @@ import * as React from "react";
  * rendered through BuildSnapshot.
  */
 export const BuildSnapshotContainer = (props) => {
+  const data = props.data;
   return (
-    <div>
-    snapshot container.
-    </div>
+    <Wrapper>
+    {data.map(snapshotData => <BuildSnapshot data={snapshotData}/>)}
```
Comment (MagoDelBlocco) [inline src/main/webapp/components/BuildSnapshotContainer.js:15]: nit:[indent by 2 spaces](https://google.github.io/styleguide/htmlcssguide.html#Indentation)

**Review by MagoDelBlocco — APPROVED** [review]
otherwise LGTM

File: src/main/webapp/components/BuildSnapshotContainer.js [hunk 2]
```diff
@@ -1,4 +1,6 @@
 import * as React from "react";
+import {Wrapper} from "./Wrapper";
+import {BuildSnapshot} from "./BuildSnapshot";
```
Comment (Somae) [inline src/main/webapp/components/BuildSnapshotContainer.js:3]: Would it make sense to order these alphabetically?

**Review by Somae — APPROVED** [review]
LGTM, added a question.

File: src/main/webapp/components/BuildSnapshotContainer.js (hunk 2 above)
Comment (samuelsogbesan) [reply src/main/webapp/components/BuildSnapshotContainer.js:3]: Perhaps - I'll go ahead and reorder them before merging.

**Pull request merged**